### DISCUSSION TITLE
Accidentally uploaded several images to github

### DISCUSSION THREAD

**Discussion by CaledoniaProject** [community/community#20576] [Other Feature Feedback, Questions, & Ideas]
If I accidentally dragged an image to the editor in "Issues" page, and I removed the `![xxx](xxx)` line, will github remove the uploaded image later?

**Comment by radar-blip** [comment]
Not exactly sure how that works but I wouldn't worry about it. 
If you removed the linkage to it, the image is probably already long gone. 👍🏼

**Reply by CaledoniaProject** [reply]
The image is still accessible when I remove the markdown text.

**Reply by radar-blip** [reply]
Is it possible for you to delete the entire issue to get rid of images?

**Reply by CaledoniaProject** [reply]
Because I don't know the behavior, so I posted the question here and expect the github stuff to answer it ...

**Reply by radar-blip** [reply]
I would imagine someone will be along soon to answer your question. 
In the meantime, here is another place you could ask the same question (if you want).
https://github.community/
